impl/src/parse.rs

```diff
@@ -96,17 +99,33 @@ impl Parse for TraitArgs {
         input.parse::<Token![=]>()?;
         let tag: LitStr = input.parse()?;
         if input.is_empty() {
-            return Ok(TraitArgs::Internal { tag });
+            return Ok(TraitArgs::Internal { tag, default_variant: None });
         }
         input.parse::<Token![,]>()?;
         if input.is_empty() {
-            return Ok(TraitArgs::Internal { tag });
+            return Ok(TraitArgs::Internal { tag, default_variant: None });
+        }
+        if input.peek::<>(kw::default_variant) {
```

Comment:
Please format with rustfmt:
```suggestion
        if input.peek(kw::default_variant) {
```